src/providers/aws.rs

```diff
@@ -969,130 +970,48 @@ impl RegionLauncher {
         &mut self,
         max_wait: Option<time::Duration>,
     ) -> Result<(), Report> {
-        tracing::info!("waiting for instances to spawn");
+        tracing::info!("waiting for {} instances to spawn", self.region.name());
```

Comment:
The region name should probably be included through the use of a span (which I think we already do). That way you don't have to tack it into every message. Maybe that span should be `info` level rather than `debug` so that it always gets included?

Reply:
I've tried this but making that span `info` doesn't seem to be enough. I think that the problem is in the instrument `level` of `launch` and e.g. `wait_for_spot_instance_requests` (`debug` and `trace`, respectively), since I can see the region in the logs after changing them to `info`. (I've reverted the change).

Reply:
Huh, that's interesting. Maybe they don't propagate spans if their level isn't met. That sounds like a bug, let me check.